src/FormattingToolbar/LinkComponent/index.js

```diff
@@ -59,7 +59,7 @@ const calculateLinkPopupPosition = (editor, openSetLink, setLinkFormPopup) => {
   return {
     // Disable semantic ui popup placement by overriding `transform`
     // and use our computed `top` and `left` values
-    popupStyle: { top, left, transform: 'none' },
+    popupStyle: { top, left, transform: 'none', width:'400px' },
```

Comment:
Unsure we want this, could you explain why this was added?

Reply:
> I think the sliding icons are too elaborate. When a user clicks on an existing link, it should look the same as creating a new link, except with a link the user can click on that opens the link in a new tab. @jeromesimeon what do you think?
> 
> ![image](https://user-images.githubusercontent.com/40569793/75689587-6ee75100-5c6f-11ea-9518-2b74afd4a52e.png)
> 
> The link could be truncated like Google
> ![image](https://user-images.githubusercontent.com/40569793/75690120-50ce2080-5c70-11ea-98d1-1a28af3827b6.png)
> 
> Can we also make the link editor box 400px wide to accommodate longer links?

@Michael-Grover wanted it to be `400px` wide so I added XD.

Reply:
OH didn't see that 👀